### ISSUE TITLE
Package is not compatible with latest rxjs ^6.0.0

### ISSUE THREAD

**Issue by 4nd1m4n** [Promact/md2#320]
#### Bug, feature request, or proposal:
- Bug.

#### What is the expected behavior?
- Correct dependency information available, that shows the user that md2 is currently using rxjs ^5.0.0 versions import pathes wich where changed in ^6.0.0.

#### What is the current behavior?
- Compilation process will ask for the rxjs-compat package wich is a compatibility layer that can just be importet and will fix all breaking changes from rxjs ^5 to ^6 so that later one can focus on porting the project 'really' from ^5 to ^6.

#### What are the steps to reproduce?
- Trying to compile a project with md2 in the webpack config.

#### What is the use-case or motivation for changing an existing behavior?
- Make md2 work again. (Simply import the rxjs-compat package for now.)

#### Which versions of Angular, MD2, OS, browsers are affected?
- Angular 6

#### Is there anything else we should know?
```
    ERROR in ./~/rxjs/Subject.js
Module not found : error : Can't resolve 'rxjs-compat/Subject' in 'X:\Y\z\src\h\XH\node_modules\rxjs'
     @ ./~/rxjs/Subject.js 6:9-39
     @ ./~/md2/tooltip/tooltip.js
     @ ./~/md2/tooltip/index.js
     @ ./~/md2/index.js
     @ dll vendor
```
- Typical error message

**Comment by eyalerez1** [comment]
Hi,
Does anybody know about a temporary workaround?

**Comment by 4nd1m4n** [comment]
> (Simply import the rxjs-compat package for now.)

Meaning in package.json:
{
  "name": "md2",
  "version": "0.0.29",
  "description": "Angular2 based Material Design components, directives and services are Accordion, Autocomplete, Chips(Tags), Collapse, Colorpicker, Data Table, Datepicker, Dialog(Modal), Menu, Multiselect, Select, Tabs, Tags(Chips), Toast and Tooltip.",
  "author": "Dharmesh Pipariya <pipariyadharmesh@gmail.com>",
  "homepage": "https://github.com/Promact/md2",
  "bugs": "https://github.com/Promact/md2/issues",
...
  "dependencies": {
    "rxjs": "^5.0.1",
...

Here you will probably have to switch to:
"rxjs": "^6.0.0",
"rxjs-compat": "^6.0.0",

Change rxjs version and add rxjs-compat. Then it -should- work again.

**Comment by eyalerez1** [comment]
Thanks!

**Comment by kevinclerc** [comment]
hey guys
How can I achieve that md2 run with angular 6x
Do I have to fork the repo and change the package.json file to the new rxjs version?

How can I then build and include the fork in my project?

**Comment by nader-eloshaiker** [comment]
Hi there, just migrated my app to Angular 6 and have this in package.json:
    "rxjs": "^6.3.3",
    "rxjs-compat": "^6.3.3",
The only thing that is not working in my app is the m2DateTimePicker?
Getting the following error:
Uncaught (in promise): TypeError: rxjs_Observable__WEBPACK_IMPORTED_MODULE_3__.Observable.fromEvent is not a function

**Comment by bababrownbear** [comment]
Can confirm, am having same issue at @nader-eloshaiker, but with colorpicker. 

![image](https://user-images.githubusercontent.com/29419183/46299734-9c563980-c567-11e8-9592-be143249909b.png)

https://github.com/ilmstudios/md2/commit/5b75cdb73dec2bfe15f45787e3f172b2135505af

**Comment by nikuelias** [comment]
Having the same issue. Any workaround?

**Comment by bababrownbear** [comment]
@nikuelias I submitted a PR: 
https://github.com/Promact/md2/pull/325

**Comment by nader-eloshaiker** [comment]
I am very appreciative of this pull request, is there any way to move this review along quicker so that is can be published? Would appreciate it even more :)

**Comment by kevinclerc** [comment]
Can someone please review this PR? this would be great!

**Comment by nader-eloshaiker** [comment]
@ilmstudios would you consider perhaps publishing your repo to npm, I get the feeling that this repo is either abandoned or relegated to casually supported.

**Comment by kevinclerc** [comment]
@ilmstudios yes please publish your repo, I think a lot of people would need this change...

**Comment by nader-eloshaiker** [comment]
Ok I’m abondoning this project and moving on. It looks like the future of this project is dead. Thanks for all the work up to this point and all the best.

**Comment by liuy97** [comment]
I republished a npm package (https://www.npmjs.com/package/angular-md2) supporting rxjs6, If someone reaches here and still waits for a new version of this project. I only used md2 datepicker, and didn't test the others.
